src/calculateConfig.ts

```diff
@@ -0,0 +1,116 @@
+import snakeCase from 'lodash/snakeCase';
```

Comment:
I've mentioned this elsewhere, but I think that this calculation ought to be done on demand within the `ApiClient.iterateEndpoints()` and `ApiClient.iterateScanFindings()` methods in the client.

I realize that this is a pattern that @aiwilliams has implemented in the Qualys, which you've been using as a guide to some extent for this integration, but since these values are each used in just one API call in the client, I think it's safer/cleaner to implement at the point of usage.

Reply:
Yep makes perfect sense, I've addressed this in this [comment](https://github.com/JupiterOne/graph-malwarebytes/pull/1#discussion_r546782426)!